ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -329,3 +329,19 @@ def qr(input, /, *, full_matrices=False, name=None):
 )
 def inv(input, adjoint=False, name=None):
     return ivy.inv(input, adjoint=adjoint)
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes(
+    {
+        "2.13.0 and below": (
+            "float32",
+            "float64",
+            "half",
+            "complex64",
+            "complex128",
+        )
+    },
+    "tensorflow",
+)
+def sqrtm(input, name=None):
+    return ivy.sqrtm(input)
```

Comment:
I think the previous reviewer mentioned that there is no `ivy.sqrtm` function in Ivy. You'll have to implement it as a composition of other ivy functions. In the other case, you may also propose this function to be added to the API, more info here:[ https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#frontend-apis]. 

If you choose to go with the latter case, I'd suggest you close this PR and work on other any other open tasks. Thanks! 🙂